### ISSUE TITLE
Defined Keyboard Chords are Printed as Inversion

### ISSUE THREAD

**Issue by spakarl** [ChordPro/chordpro#321]
## Description
I'm using the define directive to define some chord. This chord is then printed in a different inversion. I'd like to see the chord exactly as I defined it. This helps me in writing my songs in a easy playable way with the correct inversions used.

### To Reproduce
```
{title: No Inversions}
{define C7 keys 0 4 7 10}

[C7]
```
[_Test - Keyboard - C.pdf](https://github.com/ChordPro/chordpro/files/13325940/_Test.-.Keyboard.-.C.pdf)

### Expected behavior
I would like to see the chord in the correct inversion. This means it should start with the C on the left side. 

## System information
 - OS: Windows 11
 - Where did you get ChordPro from? Github
 - ChordPro version:  ChordPro core 6.030

**Comment by sciurius** [comment]
I cannot reproduce this. Can you check Help > [x] Debug in PDF and share the resultant pdf?

**Comment by spakarl** [comment]
> I cannot reproduce this. Can you check Help > [x] Debug in PDF and share the resultant pdf?

With the GUI-Tool I couldn't reproduce it either. But I'm using the command line version of chordpro:

```
chordpro --define=instrument.type=keyboard --config=chordpro.json --output="Test.pdf" --transpose=0 --generate=PDF --diagrams=user --vertical-space=7 Test.cho
```

[Test.cho](https://github.com/ChordPro/chordpro/files/13326011/Test.cho.txt)
[chordpro.json](https://github.com/ChordPro/chordpro/files/13326012/chordpro.json)

**Comment by sciurius** [comment]
It happens when you set `pdf.kbdiagrams.base` to `"F"` instead of `"C"`. 
I'll check it out.

**Comment by sciurius** [comment]
Fixed in 6.030_023.

**Comment by spakarl** [comment]
> Fixed in 6.030_023.

I can't find a download link. Is it available already?
